indexer-common/src/domain/ledger/ledger_state.rs

```diff
@@ -610,4 +626,36 @@ mod tests {
         let intent_hash2 = compute_claim_rewards_intent_hash_v6(&owner, value, &nonce);
         assert_eq!(intent_hash, intent_hash2);
     }
+
+    #[test]
+    fn test_make_unshielded_utxos_v6() {
+        let network_id = NetworkId::Undeployed;
+
+        let transaction = StandardTransactionV6 {
+            network_id: network_id.to_string(),
+            intents: Default::default(),
+            guaranteed_coins: Default::default(),
+            fallible_coins: Default::default(),
+            binding_randomness: EmbeddedFr::from_le_bytes(&[0u8; 32]).unwrap(),
+        };
+        let ledger_transaction = TransactionV6::Standard(transaction);
+
+        let ledger_state = LedgerStateV6::new(network_id);
+
+        let (created, spent) = make_unshielded_utxos_v6(
+            ledger_transaction.clone(),
+            &TransactionResult::Failure,
+            &ledger_state,
+        );
+        assert!(created.is_empty());
+        assert!(spent.is_empty());
+
+        let (created, spent) = make_unshielded_utxos_v6(
```

Comment:
@cosmir17 , what's the purpose of this test? Successful transactions could have created/spent UTXOs (of course this empty test transaction does not).